data-processing/entities/citations/commands/resolve_bibitems.py

```diff
@@ -92,6 +94,10 @@ def process(self, item: MatchTask) -> Iterator[BibitemMatch]:
                     bibitem.id_,
                     item.arxiv_id,
                 )
+        if item.bibitems and ref_match_count == 0:
```

Comment:
So we'll only warn if none of them could be matched. But if at least one is matched, we won't warn. Is that right? And I guess, for my own understanding, why is matching none more serious than not matching some?

Reply:
Good question. I'm also looking to get more visibility into `no_citations` error. With this, we can know whether that error is caused by scholarphi not being able to extract any reference vs being able to extract reference but can't match it. Reading this, I made an update to the code to better suit that. I also updated the PR description.

![Screen Shot 2021-04-16 at 11 02 38](https://user-images.githubusercontent.com/52424731/115065672-4508e200-9ea3-11eb-8983-0750a63a317f.png)